src/main.js

```diff
@@ -13,6 +13,9 @@ const TrashRouteHandler = require('./routes/trash');
 const TrashcanRouteHandler = require('./routes/trashcan');
 const ScoreRouteHandler = require('./routes/score');
 
+// Use environment port specification if set; if not, use port 8080 (makes app compatible with heroku)
+const PORT = process.env.PORT || 8080
```

Comment:
use `Number.parseInt` for the env var, it is string otherwise.

Reply:
fixed!